kitchen.yml

```diff
@@ -52,13 +53,24 @@ suites:
       nodejs:
         npm:
           install_method: source
-  - name: npm_packages
-    run_list: recipe[nodejs]
+  - name: package
+    run_list:
+      - recipe[test::default]
+  - name: source
+    run_list:
+      - recipe[test::source]
+  - name: options
+    includes:
+      - ubuntu-20.04
```

Comment:
Why only this platform?

Reply:
I think b/c `npm` works the same regardless of platform, so we only need to test it once, yes? I.e., the intent behind this test suite is to validate `npm_package`’s new `package_options` property, while `npm_package` working well on all platforms is validated by the `package` suite.

@MarkGibbons, it may make sense to combine the two suites, to reduce the number of suites. I.e., you can get two birds with one stone, by setting:

```yaml
- name: package
  run_list:
    - recipe[test::default]
  attributes:
    nodes:
      packages:
        - nodejs
        - acpid
        - adcli
        package_options:
          acpid: "--force-yes"
        package_action:
          adcli: :nothing
```

Reply:
Sometimes you can run into odd platform specific problems so it's always nice to test them on all just in case you catch something.

Reply:
I'm quite willing to test everything.  But as I've added test I've found a lot of stuff that just doesn't work.  I'm using #218 as a ticket to track the things that need to be fixed and added back.

Reply:
Are they working only on specific suites and not on others? If so I'd exclude them on the broken ones (including a comment for why). You can see how to do that in the percona cookbook.

Reply:
I added centos-7 back to the list of OSs.